packages/core/content-manager/admin/src/hooks/useDocument.ts

```diff
@@ -202,11 +202,8 @@ const useDoc = () => {
     }
   );
 
-  const singleTypeId = collectionType === SINGLE_TYPES ? document.document?.documentId : undefined;
+  const returnId = origin || id === 'create' ? undefined : id;
```

Comment:
I'm not sure I understand thise change ? for a single type this will return undefined ? 

did you remove it because it's spread below and will replace the id (document has an id) ? if so I would suggestion removing this line entirely and just spread the document below 🤔

Reply:
If you compare to `releases/5` you'll see the logic for the return id here hasn't actually changed 

What we did with `const singleTypeId = collectionType === SINGLE_TYPES ? document.document?.documentId : undefined;` was an attempt to fix permissions with single types but we found it had too many side effects

These InputRenderer changes are what we've done instead https://github.com/strapi/strapi/blob/ca278734871c5ea02ab7d1bbe932089ac2bd16be/packages/core/content-manager/admin/src/pages/EditView/components/InputRenderer.tsx#L49-L52